docs/source/forces.md

```diff
@@ -205,6 +205,36 @@ The following snippet defines a plane that acts on the whole system and will not
 
 ````
 
+## Attraction plane
+
+This kind of external force implements a attraction plane that constrains particles to stay on one side of it while attracting them with a constant force. The repulsion is implemented as a harmonic interaction, but the stiffness can be made arbitrarily high to mimic a hard repulsion.
+
+A force of this kind is specified with `type = attraction_plane`. The relevant keys are:
+
+* `particle = <int>`: comma-separated list of indices of particles to apply the force to. A value of `-1` or `all` applies it to all particles. Entries separated by a dash "-" get expanded in a list of all the particles on a same strand comprised between the two indices. For instance, `particle = 1,2,5-7` applies the force to 1,2,5,6,7 if 5 and 7 are on the same strand.
+* `stiff = <float>`: stiffness of the repulsion.
```

Comment:
I guess stiffness of the attraction?

Reply:
I tried to highlight in the documentation that the same value "stiff" is used for both the attractive and the repulsion part. If necessary I could also expose it as 2 separate input values.